clarifai/client/input.py

```diff
@@ -729,6 +731,46 @@ def delete_inputs(self, inputs: List[Input]) -> None:
       raise Exception(response.status)
     self.logger.info("\nInputs Deleted\n%s", response.status)
 
+  def download_inputs(self, inputs: List[Input]) -> List[bytes]:
+    """Download list of input objects from the app.
```

Comment:
Shouldn't we store inputs as files, instead of returning `bytes` ?

Reply:
This is for the user to use the file as per their need, they can use it in the runtime or choose to save the file to the local. Like how we have in model prediction..